core/tests/protractor_utils/workflow.js

```diff
@@ -31,6 +31,29 @@ var createExploration = function() {
   explorationEditorMainTab.exitTutorial();
 };
 
+var setTitleForExploration = function(name) {
+  element(by.model('explorationTitleService.displayed')).clear().then(
+    function() {
+      element(by.model('explorationTitleService.displayed')).sendKeys(name);
```

Comment:
@amey-kudari, Do you see this pattern anywhere for writing the e2e test? I think for e2e you should not consider the implementation details just try adding the title to the element instead of angularService.

I'll highly suggest you to use `ExplorationEditorSettingsTab.setTitle` instead of adding duplicate code.

Reply:
Alright, I agree I changed it.